benchmarks/compressed_scan.cpp

```diff
@@ -316,8 +316,11 @@ struct vector_scan {
         read_ptr += 3 * (OUTPUT_ELEMENTS_PER_VECTOR)*9 / 8;
 
 #if CLANG_COMPILER
```

Comment:
@He3lixxx we can remove this check. the generic `#pragma unroll` also works with GCC.